Source/Entities/Views/EntityViewBase.generated.cst

```diff
@@ -240,18 +241,36 @@ namespace <%=NameSpace%>
 			set { this._isNew = value; }
 		}
 
+		#endregion
+
+		#region View Meta Data
+
 		/// <summary>
-		///		The name of the underlying database table.
+		///	The name of the underlying database view.
 		/// </summary>
 		[BrowsableAttribute(false), XmlIgnoreAttribute()]
 		public string ViewName
 		{
 			get { return "<%=SourceView.Name%>"; }
 		}
-
-		
-		#endregion
 		
+		/// <summary>
+		///	The name of the underlying database view's columns.
+		/// </summary>
+		[BrowsableAttribute(false), XmlIgnoreAttribute()]
+		public string[] ViewColumns

```

Comment:
I'm really not sure why we'd want to ever expose this.

Reply:
It currently exposed for tables, just not views, this just brings views in line with how tables are generated.
Also, I use it in reflection when I need to know the original name of the database column before it goes through NetTiers normalization process.  I think I would prefer some sort of dictionary mapping, and maybe some day i'll do that, but for now copying the way tables are handled was enough.